services/libs/common/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@crowd/logging": "file:../logging",
     "@crowd/types": "file:../types",
+    "lodash": "~4.17.21",
```

Comment:
Lodash is a huge library ~600kb and just loading everything is a pain for nodejs as well. 
Two options here:
- functionality can be rewritten without it if possible
- use dedicated function npm packages like for instance lodash merge https://www.npmjs.com/package/lodash.merge